src/basic/chase.c

```diff
@@ -103,6 +103,9 @@ int chaseat(
         if (isempty(path))
                 path = ".";
 
+        if (dir_fd == AT_FDCWD)
```

Comment:
We can also check if `dir_fd_is_root()` is true and skip the flag as well in that case.